source/_integrations/weatherflow.markdown

```diff
@@ -22,6 +22,13 @@ The **WeatherFlow** {% term integration %} is a local-only {% term integration %
 You may see slight deviations between the values reported in Home Assistant and the values in the WeatherFlow App. This is because the WeatherFlow app considers both forecasts and neighboring weather stations in addition to the local data used in this {% term integration %}.
 {% endnote %}
 
+### Which integration should I use
+
+The [WeatherFlow](https://www.home-assistant.io/integrations/weatherflow) integration provides local communications with your weather station via UDP. It returns raw sensor data readings.
+The [WeatherFlow Cloud](https://www.home-assistant.io/integrations/weatherflow_cloud) integration provides a data feed reflective of what is presented in the Tempest mobile application - including Forecast data. WeatherFlow explains the differences in their FAQ:
+
+> Can I access data from the hardware locally? To ensure access to the best data, all third-party applications and integrations should use the remote interfaces (REST & Websockets) as their primary source for data, even if they are running on the same network as the local Tempest device. There is also a local UDP interface available for those applications that require completely off-grid applications, but this should only be used as a backup to the remote interfaces.
```

Comment:
If this is intended as a FAQ rather than a quote then I would display this differently.

Reply:
How do you suggest I change it up?

Reply:
Just a header (question) with the paragraph (answer) underneath. Furthermore, I would describe the answer as straightforwardly as possible.

./Klaas

Reply:
This seem better? (tried to simplify it)

<img width="477" alt="image" src="https://github.com/user-attachments/assets/0f104ef6-71db-4efd-ac8e-b0544d3db848">

Reply:
I think I'm going with this one :)